avaframe/com1DFA/DFAfunctionsCython.pyx

```diff
@@ -1190,58 +1262,71 @@ def computeGradC(cfg, particles, headerNeighbourGrid, headerNormalGrid, double[:
         for p in range(iPstart, iPend):
             # index of particle in neighbour box
             l = partInCell[p]
-            if j != l:
+            if k != l:
                 dx = xArray[l] - x
                 dy = yArray[l] - y
                 dz = zArray[l] - z
-                if SPHoption == 1:
-                  dz = 0
-                  # get norm of r = xj - xl
-                  r = norm(dx, dy, dz)
-                  if r < minRKern * rKernel:
-                      # impose a minimum distance between particles
-                      # dx = minRKern * rKernel * dx
-                      # dy = minRKern * rKernel * dy
-                      r = minRKern * rKernel
-                  if r < rKernel:
-                      hr = rKernel - r
-                      dwdr = dfacKernel * hr * hr
-                      mdwdrr = mass[l] * dwdr / r
-                      gradhX = gradhX + mdwdrr*dx
-                      gradhY = gradhY + mdwdrr*dy
-                      gradhZ = gradhZ + mdwdrr*dz
-                      gravAcc3 = gravAcc
 
-                if SPHoption == 2:
-                  # get coordinates in local coord system
-                  r1 = scalProd(dx, dy, dz, ux, uy, uz)
-                  r2 = scalProd(dx, dy, dz, uxOrtho, uyOrtho, uzOrtho)
-                  # impse r3=0 even if the particle is not exactly on the tengent plane
-                  # get norm of r = xj - xl
-                  r = norm(r1, r2, 0)
-                  if r < minRKern * rKernel:
-                      # impose a minimum distance between particles
-                      r1 = minRKern * rKernel * r1
-                      r2 = minRKern * rKernel * r2
-                      r = minRKern * rKernel
-                  if r < rKernel:
-                      hr = rKernel - r
-                      dwdr = dfacKernel * hr * hr
-                      mdwdrr = mass[l] * dwdr / r
-                      G1 = mdwdrr * K1*r1
-                      G2 = mdwdrr * K2*r2
+#----------------------------SPHOPTION = 1--------------------------------------
```

Comment:
Maybe add a one liner below each comment with the same info as in the ini. I.e. what each option does